tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -738,6 +740,19 @@ public int getRequiredVotes() {
         return requiredVotes;
     }
 
+    /**
+     * What are the adjusted required votes for this item?
```

Comment:
```suggestion
     * What are the adjusted required votes for this item, taking into account the trunkStatus?
```

Reply:
I would also update getRequiredVotes to say that it _only_ takes the xpath and locale into account. 

`canFlagOnLosing()` wasn't updated, should it be? 

Perhaps this function should be `getRequiredVotes()` as this is the _actual_ required votes, and the other one renamed to `getUnadjustedRequiredVotes()`

I'm splitting hairs here, but this is critical core machinery, and can be challenging to trace through. ('Adjusted' doesn't tell me much about what is happening, and I know we have several 'adjusted' functions in various parts now)

Reply:
canFlagOnLosing works correctly, as it stands, since requiredVotes == HIGH_BAR even when getAdjustedRequiredVotes may return LOWER_BAR

I like your suggestions -- the "adjusted" business has a disagreeable smell! there's definitely a lot of room for improvement in the implementation and naming and comments. I was trying to avoid touching/changing more of the code for this PR than was really necessary to accomplish the goal of this ticket; maybe too cautious...

Reply:
actually maybe I was wrong about canFlagOnLosing -- I'll email cldr-core2 for policy clarification